applet/src/gfx.c

```diff
@@ -237,6 +237,10 @@ void gfx_drawbmp_hook( void *bmp, int x, int y )
         return ;
     }
     gfx_drawbmp( bmp, x, y );
+      // redraw promiscuous mode eye icon overlapped by antenna icon on MD390/G             20161118 - DL2MF
+    if(  ( global_addl_config.promtg ) && ( y == 0 )) {
```

Comment:
stop putting date and callsign in patches. if everybody did that it becomes a mess.